app/components/support_interface/provider_type_tag_component.rb

```diff
@@ -0,0 +1,33 @@
+module SupportInterface
+  class ProviderTypeTagComponent < ViewComponent::Base
+    def initialize(provider:)
+      @provider_type = provider.provider_type
+    end
+
+    def render?
+      @provider_type.present?
+    end
+
+    def text
+      case @provider_type
+      when 'lead_school'
+        'School Direct'
+      when 'scitt'
+        'SCITT'
+      else
+        'HEI'
+      end
+    end
```

Comment:
can we map these to constants for simplicity and to remove the case statement

e.g. something like...
```ruby
 PROVIDER_TYPE_TEXT = { lead_school: 'School Direct',
                        scitt: 'SCITT' }


def text 
   PROVIDER_TYPE_TEXT.fetch(@provider_type.to_sym, 'HEI')
end
```